### DISCUSSION TITLE
Yi Dome Guard IFUS with fw ver. 9.0.05.33

### DISCUSSION THREAD

**Discussion by okroypl** [roleoroleo/yi-hack-Allwinner-v2#354] [Ideas]
I Wonder to run correctly yi hack on
Yi Dome Guard IFUS with fw ver. 9.0.05.33
I have install r35 
I can connect to web site  I can controll camera moves but no video on web site and on rstp

**Comment by ghost** [comment]
@okroypl Are you sure it's not r30gb?

**Comment by okroypl** [comment]
My camera is" dome guard "not "dome"
Schould i test r30?

śr., 16 lut 2022, 01:58 użytkownik 1-Al ***@***.***> napisał:

**Reply by ghost** [reply]
> My camera is" dome guard "not "dome" 

Oh right, I didn't notice that. Looks like the readme needs updating. 

![dg](https://user-images.githubusercontent.com/94827997/154610774-3313080a-34cd-4eff-82f4-edbfac208a44.png)

I have an r30gb and on the bottom it say's "YI Dome Guard Model:YRS.3019"


> Schould i test r30?

You should find your model by checking your mtdblock0 file from your backup by searching for '/uboot' and seeing what comes after it.

**Comment by Czester0** [comment]
> I Wonder to run correctly yi hack on Yi Dome Guard IFUS with fw ver. 9.0.05.33 I have install r35 I can connect to web site I can controll camera moves but no video on web site and on rstp

I did exactly the same same camera with the same software. There was no video broadcast.
But 1-AI said well: see to mtdblock0.bin For me it was r30gb.
I threw r30gb into my YRS.3019 and the image worked.

Thank you.

**Comment by okroypl** [comment]
I have tested and IT is working but camera is restarting in 30seconds. IS
previous r35 change something on camera files? Should i make restore from
backup?


pt., 25 lut 2022, 12:07 użytkownik Czester0 ***@***.***>
napisał:

**Reply by roleoroleo** [reply]
Try enabling swap.

**Reply by Czester0** [reply]
try to turn on the audio quickly, restart then turn off the audio.

**Comment by Czester0** [comment]
After a month of use - I have such conclusions. At the first boot with the card or after Reset to yi-hack default, the camera works rather well and for a long time. But when in my case I want to turn on mqtt, the camera reboots quite often, usually every 10-15 minutes. I tried to switch cards. Tried turning on Swap File on SD card - it doesn't help. My YI Dome Guard /YRS.3019 camera shows the mtdblock0 file points to the r30_gb file

**Reply by roleoroleo** [reply]
> But when in my case I want to turn on mqtt, the camera reboots quite often, usually every 10-15 minutes. I tried to switch cards. Tried turning on Swap File on SD card - it doesn't help.

Normally this problem disappears when you enable swap file.
The cam crashes because it tries to take a snapshot and the available memory is not enough.
Check if the swap file is ok:
```
root@yi-hack-allwinner-v2:~# . /tmp/sd/yi-hack/script/env.sh
root@yi-hack-allwinner-v2:~# free
              total        used        free      shared  buff/cache   available
Mem:          60912       40252       12688        1860        7972       17776
Swap:         65532        2744       62788
```

Or you could try to leave empty the field "Topic Suffix for jpeg image".

> My YI Dome Guard /YRS.3019 camera shows the mtdblock0 file points to the r30_gb file

I don't understand.

**Reply by Czester0** [reply]
When I started with the YI hack 0.2.0 later 0.2.1 there were all the time problems with reboots. At 0.2.1 on average every 15 minutes, at 0.2.0 about a couple of hours. Of course, with sd swap enabled. It looks like 0.1.9 is much better for this camera because it has been testing for 2 days without rebooting on 3 cameras. In fact, there is not much memory.
```
root@YI2:~# free
              total        used        free      shared  buff/cache   available
Mem:          60908       43000        1468        1780       16440       15104
Swap:         65532        3732       61800
```
